Build.fsx

```diff
@@ -115,14 +116,14 @@ Target "TestOnly" (fun _ ->
     let nunit2TestAssemblies = !! (sprintf "Src/AutoFixture.NUnit2.*Test/bin/%s/*Test.dll" configuration)
 
     nunit2TestAssemblies
-    |> NUnit (fun p -> { p with StopOnError = false
-                                OutputFile = "NUnit2TestResult.xml" })
+    |> NUnitSequential.NUnit (fun p -> { p with StopOnError = false
+                                                OutputFile = "NUnit2TestResult.xml" })
 
     let nunit3TestAssemblies = !! (sprintf "Src/AutoFixture.NUnit3.UnitTest/bin/%s/Ploeh.AutoFixture.NUnit3.UnitTest.dll" configuration)
 
     nunit3TestAssemblies
-    |> NUnit3 (fun p -> { p with StopOnError = false
-                                 ResultSpecs = ["NUnit3TestResult.xml;format=nunit2"] })
+    |> Testing.NUnit3.NUnit3 (fun p -> { p with StopOnError = false
+                                                ResultSpecs = ["NUnit3TestResult.xml;format=nunit2"] })
```

Comment:
This change seems to be unrelated to the new `AppVeyor` target. Could you please commit it separately and describe its purpose in the message?